iothub/service/src/RegistryManager.cs

```diff
@@ -656,10 +667,20 @@ public void Dispose()
         /// <param name="importBlobContainerUri"></param>
```

Comment:
Shouldn't this be InputBlobContainer?

Reply:
It probably should have been, but I cannot rename it now in case some customer used named parameters.